docs/latest/user/rate-limit.md

```diff
@@ -571,7 +571,17 @@ transfer-encoding: chunked
 
 * The default installation of Envoy Gateway installs a default [EnvoyGateway][] configuration and provides the initial rate
 limit kubernetes resources settings. such as `replicas` is 1, requests resources cpu is `100m`, memory is `512Mi`. the others
-like container `image`, `securityContext` and pod `annotations` and `securityContext` can be modified by modifying the `ConfigMap`.
+like container `image`, `securityContext`, `env` and pod `annotations` and `securityContext` can be modified by modifying the `ConfigMap`.
+
+* The default [EnvoyGateway][] configuration and provides some initial `env` settings. 
+rate limit `env` setting reference https://github.com/envoyproxy/ratelimit/blob/main/src/settings/settings.go
```

Comment:
lets rm this, this is hard to track since it might keep changing

Reply:
done